lib/rspec/buildkite/analytics/reporter.rb

```diff
@@ -18,6 +18,7 @@ def handle_example(notification)
 
       if trace
         trace.example = example
+        trace.failure_reason, trace.failure_expanded = failure_info(notification) if example.execution_result.status == :failed
```

Comment:
Shorter line length? Not overly fussed.

```suggestion
        if example.execution_result.status == :failed
          trace.failure_reason, trace.failure_expanded = failure_info(notification)
        end
```

Reply:
🤷‍♀️ I'll leave it for the rubocop PR